src/node_url.cc

```diff
@@ -1653,9 +1653,10 @@ void URL::Parse(const char* input,
               state = kFragment;
               break;
             default:
-              if ((!IsWindowsDriveLetter(ch, p[1]) ||
-                   end - p == 1 ||
-                   (p[2] != '/' &&
+              if ((end - p == 0 ||
```

Comment:
The **remaining** means after *ch*, so *p[0]* must not be counted here. Correct to `end - (p+1) == 0` or just `end - p == 1`

There is an example, which explains the term **remaining**: https://url.spec.whatwg.org/#remaining